frontend/src/index.tsx

```diff
@@ -11,7 +11,7 @@ ReactDOM.render(
   document.getElementById("root")
 );
 
-// If you want to start measuring performance in your app, pass a function
+// If you want to start measuring performance in your frontend, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
 reportWebVitals();
```

Comment:
I think we can just remove this, we don't use it and don't plan to, as far as I'm aware.

Reply:
this part was made by Jon so I was planning on not touching it haha